examples/drupal-mysql/README.md

```diff
@@ -0,0 +1,57 @@
+## Drupal / MySQL example
+
+This example shows how to deploy a multi-tier Drupal web app using Apollo and Marathon Mesos framework.
+
+The example includes -
+
+* A web frontend (Using Drupal Commerce Kickstart distribution)
+* A MySQL (5.5) backend
+
+We'll use the [mysql](https://registry.hub.docker.com/_/mysql/) official [Docker](https://www.docker.com/) image and an [apollo-commerce-kickstart](https://registry.hub.docker.com/u/capgemini/apollo-commerce-kickstart/) image which contains [php-apache](https://registry.hub.docker.com/_/php/), Drush and [Commerce Kickstart](https://www.drupal.org/project/commerce_kickstart).
+
+The examples are to be deployed on top of the Marathon Mesos framework.
+
+### Start the Drupal and MySQL services
+
+First, edit ```drupal-mysql.json```, in the mysql app service definition, to use database credentials that you specify (or leave as is if you are happy and this is just for demo purposes).
+
+If you changed the database credentials as above, then edit the drupal app definition in ```drupal-mysql.json```, to use database credentials that you specify (or leave as is if you are happy and this is just for demo purposes).
+
+Start the services by running the following command (from somewhere you can access the REST interface of Marathon). Replace $MARATHON_IP with the actual hostname/ip address of the marathon instance.
+Marathon by default should be deployed in the ```mesos_master``` nodes.
+
+```
+curl -X POST -HContent-Type:application/json -d @drupal-mysql.json $MARATHON_IP:8080/v2/groups
+```
```

Comment:
We could include a comment on how to delete it -> curl -XDELETE  $MARATHON_IP:8080/v2/groups/drupal-mysql-example